pkg/interface/link/src/js/components/link.js

```diff
@@ -198,7 +197,7 @@ export class LinkDetail extends Component {
             commentPage={props.commentPage}
             members={props.members}
             popout={props.popout}
-            url={props.data.url}
+            url={props.url}
             linkPage={props.page}
             linkIndex={props.link}
```

Comment:
And here (and why do these have different names? I wish I got to refactor this.)